src/SerEvaluator.cs

```diff
@@ -645,7 +645,7 @@ private SerConfig GetNewJson(UserParameter parameter, string userJson, string wo
 
                     report["connections"] = new JArray(newUserConnections);
                     var distribute = report["distribute"];
-                    var children = distribute.Children().Children();
+                    var children = distribute?.Children().Children().ToList() ?? new List<JToken>();
```

Comment:
@mossbeard are you shure that this never crashes? what Happens if there are no childen under the children?

Reply:
@konne thank you for the tip.
I have fixed the code (new PR).
It was tested with emtpy and no existing distribute property.